docs/components/motor/_index.md

```diff
@@ -41,6 +41,13 @@ Model | Description <a name="model-table"></a>
 [`roboclaw`](./roboclaw/) | [Standard brushed DC motor](https://en.wikipedia.org/wiki/DC_motor) driven by [Basicmicro's](https://www.basicmicro.com/) [RoboClaw](https://www.basicmicro.com/RoboClaw-2x30A-Motor-Controller_p_9.html) motor controller
 [`fake`](./fake/) | Used to test code without hardware
 
+Viam also provides the following motor models as [modular resources](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:odrive:canbus`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with CANbus communication
+ [`viam:odrive:serial`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with serial communication
+

```

Comment:
Let's add one sentence after the modules with a bit of next steps, if you agree. Same for other instances in this PR.

> These modules can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry)

Reply:
I'm good with that, seems like a good chance to advertise the registry! Thanks andrew!